tron/core/actionrun.py

```diff
@@ -179,6 +180,7 @@ def __init__(
         mem=None,
         service=None,
         deploy_group=None,
+        machine=None,
```

Comment:
Do you still need this?

Reply:
I added this so I could inject a mock state machine in the tests.